I'm sorry, but an uncaught exception occurred.

Compiling ATL code at game/script.rpy:175
  File "game/script.rpy", line 797, in script
    a "conversation pending"
Exception: Can't register channel outside of init phase.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 797, in script
    a "conversation pending"
  File "C:\Users\singi\Documents\renpy-7.3.5-sdk\renpy\ast.py", line 708, in execute
    renpy.exports.say(who, what, *args, **kwargs)
  File "C:\Users\singi\Documents\renpy-7.3.5-sdk\renpy\exports.py", line 1345, in say
    who(what, *args, **kwargs)
  File "C:\Users\singi\Documents\renpy-7.3.5-sdk\renpy\character.py", line 1142, in __call__
    self.do_display(who, what, cb_args=self.cb_args, **display_args)
  File "C:\Users\singi\Documents\renpy-7.3.5-sdk\renpy\character.py", line 842, in do_display
    **display_args)
  File "C:\Users\singi\Documents\renpy-7.3.5-sdk\renpy\character.py", line 591, in display_say
    rv = renpy.ui.interact(mouse='say', type=type, roll_forward=roll_forward)
  File "C:\Users\singi\Documents\renpy-7.3.5-sdk\renpy\ui.py", line 297, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "C:\Users\singi\Documents\renpy-7.3.5-sdk\renpy\display\core.py", line 2702, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, **kwargs)
  File "C:\Users\singi\Documents\renpy-7.3.5-sdk\renpy\display\core.py", line 3094, in interact_core
    root_widget.visit_all(lambda i : i.per_interact())
  File "C:\Users\singi\Documents\renpy-7.3.5-sdk\renpy\display\core.py", line 541, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\singi\Documents\renpy-7.3.5-sdk\renpy\display\core.py", line 541, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\singi\Documents\renpy-7.3.5-sdk\renpy\display\core.py", line 541, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\singi\Documents\renpy-7.3.5-sdk\renpy\display\core.py", line 541, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\singi\Documents\renpy-7.3.5-sdk\renpy\display\core.py", line 541, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\singi\Documents\renpy-7.3.5-sdk\renpy\display\core.py", line 531, in visit_all
    for d in self.visit():
  File "C:\Users\singi\Documents\renpy-7.3.5-sdk\renpy\atl.py", line 582, in visit
    block = self.compile()
  File "C:\Users\singi\Documents\renpy-7.3.5-sdk\renpy\atl.py", line 489, in compile
    block = self.atl.compile(self.context)
  File "C:\Users\singi\Documents\renpy-7.3.5-sdk\renpy\atl.py", line 684, in compile
    statements = [ i.compile(ctx) for i in self.statements ]
  File "C:\Users\singi\Documents\renpy-7.3.5-sdk\renpy\atl.py", line 897, in compile
    child = ctx.eval(expr)
  File "C:\Users\singi\Documents\renpy-7.3.5-sdk\renpy\atl.py", line 246, in eval
    return eval(expr, renpy.store.__dict__, self.context)  # @UndefinedVariable
  File "<string>", line 1, in <module>
  File "C:\Users\singi\Documents\renpy-7.3.5-sdk\renpy\display\video.py", line 369, in __init__
    self.ensure_channel(self.channel)
  File "C:\Users\singi\Documents\renpy-7.3.5-sdk\renpy\display\video.py", line 340, in ensure_channel
    renpy.audio.music.register_channel(name, renpy.config.movie_mixer, loop=True, stop_on_mute=False, movie=True, framedrop=framedrop)
  File "C:\Users\singi\Documents\renpy-7.3.5-sdk\renpy\audio\audio.py", line 757, in register_channel
    raise Exception("Can't register channel outside of init phase.")
Exception: Can't register channel outside of init phase.

Windows-8-6.2.9200
Ren'Py 7.3.5.606
AE Party 1.0
Wed Aug 26 18:21:13 2020
